### ISSUE TITLE
Colaborar

### ISSUE THREAD

**Issue by gustavo-oliveira-rosa** [seade-R/dados-covid-sp#19]
Boa noite, gostaria de ser um colaborador do projeto.

Como eu poderia ajudar este respositorio e me tornar um dos manteiners?

**Comment by leobarone** [comment]
Obrigado pela disposição, @KoenomatachiSan . Infelizmente, não é possível aceitar colaboradores externos neste projeto.

**Comment by gustavo-oliveira-rosa** [comment]
Claro! Agradeço a resposta. Estou a disposição caso essa regra mude. :)
